### ISSUE TITLE
Audit list priorities - all purple...

### ISSUE THREAD

**Issue by kusler** [GoogleChrome/web.dev#509]
The fact that high, medium and low priority are all purple throws me... I know that we're trying to keep the green, yellow, red as the 'slow, average, fast' - but there needs to be some sort of demarkation that indicates hierarchy color wise here... Do we agree?

<img width="1680" alt="screen shot 2018-12-03 at 3 17 39 pm" src="https://user-images.githubusercontent.com/40670445/49407871-fbbefa00-f70e-11e8-9ce4-6a7239cb9327.png">

**Comment by hwikyounglee** [comment]
Shall we try:

High –  FG  #E53935 or rgb(229, 57, 53), BG rgba(229, 57, 53, .10)
Medium –  FG #DD2C00 or rgb(221, 44, 0), BG rgba(251, 140, 0, .10)
Low –  FG #3740FF or rgb(55, 64, 255), BG rgba(55, 64, 255, .10)

![k8zxrlxlyfs](https://user-images.githubusercontent.com/7561395/49418915-579e7880-f739-11e8-8a01-f7dde6366ca0.png)

**Comment by robdodson** [comment]
I think this is a great suggestion. Regarding the colors—we'll need to make sure they meet accessibility contrast ratios. I think these (and the colors we are currently using) may not have enough contrast. You can test with this tool:

https://contrast-ratio.com/#%23E53935-on-%23FCEBEA

**Comment by robdodson** [comment]
Actually it looks like the blue colors meet the threshold, but the red and orange may need to be boosted a bit. I was just using the eye dropper on the image because the tool I was using didn't seem to like rgba

**Comment by hwikyounglee** [comment]
Bumped up the contrasts being over 4.5, and updated colors with hex.

**Comment by robdodson** [comment]
Patch is at cl/224224651. Will close once it lands.

**Comment by robdodson** [comment]
Should be fixed in source now. It'll roll out next time DevSite does a release.
